### PR TITLE
Backend for token validator

### PR DESCRIPTION
**Pull request by vad33r**
This PR add a backend for the token validator.

### CODE REVIEW

File: eox_tenant/edxapp_wrapper/backends/oauth_dispatch_j_v1.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+""" Backend abstraction. """
+
+from openedx.core.djangoapps.oauth_dispatch.dot_overrides.validators import (  # pylint: disable=import-error
+    EdxOAuth2Validator,
+)
+
+def get_edx_oauth2_validator_class():
+    """ Backend to get EdxOAuth2Validator class
```
Comment (andrey-canon) [inline eox_tenant/edxapp_wrapper/backends/oauth_dispatch_j_v1.py:9]: the double quotes must be in the same line if the text has only one line

File: eox_tenant/edxapp_wrapper/oauth_dispatch.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Oauth_dispatchs definitions.
+"""
+
+from importlib import import_module
+
+from django.conf import settings
+
+
+def get_edx_oauth2_validator_class():
+    """ Backend to get EdxOAuth2Validator class
```
Comment (andrey-canon) [inline eox_tenant/edxapp_wrapper/oauth_dispatch.py:11]: the double quotes must be in the same line if the text has only one line

**Review by andrey-canon — CHANGES_REQUESTED** [review]

File: eox_tenant/edxapp_wrapper/backends/oauth_dispatch_j_v1.py (hunk 1 above)
Comment (vad33r) [reply eox_tenant/edxapp_wrapper/backends/oauth_dispatch_j_v1.py]: ty, solved

File: eox_tenant/edxapp_wrapper/oauth_dispatch.py (hunk 2 above)
Comment (vad33r) [reply eox_tenant/edxapp_wrapper/oauth_dispatch.py]: ty, solved

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**